packages/pebble-shared/svgs/house.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="1000" height="1000" viewBox="0 0 1000 1000" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
dont add duplicate or similar icons. use icon `home-filled-2` instead of home

also, check [pebble icons](https://pebble-web.netlify.app/?path=/story/theme-icons--list) for anymore issues